src/ServicePulse.Host/vue/src/views/ConfigurationView.vue

```diff
@@ -45,7 +46,7 @@ onMounted(() => {
           <h5 v-if="!licenseStatus.isExpired" :class="{ active: subIsActive('connections') }" class="nav-item">
             <RouterLink :to="{ name: 'connections' }">
               Connections
-              <template v-if="connectionState.unableToConnect || monitoringConnectionState.unableToConnect">
+              <template v-if="connectionState.unableToConnect || (monitoringConnectionState.unableToConnect && !useIsMonitoringDisabled())">
```

Comment:
Is there a useIsMonitoringEnabled method available? double negatives (not userIsMonitoringDisabled) are sometimes hard for my brain to process.

Reply:
I have just made it available. Added an inverted check.